index.html

```diff
@@ -4380,6 +4381,125 @@ <h3>Parsing content</h3>
       [[RFC2048]] and [[RFC2046]].
     </p>
   </section>
+  <section> <h3>Privacy implications and implementation considerations</h3>
+    <p>
+      The comparison to barcodes or QR codes is appropriate because NFC tags
+      are another non-human-readable method of exchanging data and sharing
+      them can have unforeseen privacy and security implications. For a web
+      site to read a QR code, a piece of interruptive UI must be used (the
+      camera) which captures an image and makes it apparent that the contents
+      of this image (including the QR code) will be available to the web page,
+      making it clear to the user that scanning is being performed.
+    </p>
+    <p>
+      Scanning a tag with NFC requires the user to place the scanning device
+      (e.g. phone) in close proximity to the NFC tag.
```

Comment:
Shall we specify how much "(usually 5-10 cm, 2-4 inches)" so that it's clear it is that close?